asset-transfer-ledger-queries/chaincode-go/go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/hyperledger/fabric-samples/chaincode/marbles02/go
```

Comment:
should this be `github.com/hyperledger/fabric-samples/asset-transfer-ledger-queries/chaincode-go`

Reply:
🛠️ Ah I copied this over from marbles02.